MUI/build.gradle

```
@@ -126,7 +135,7 @@ spotless {
         endWithNewline()
     }
     java {
-        target 'src/*/java/**/*.java', 'ghidra_scripts/*.java'
+        target 'src/*/java/mui/**/*.java', 'ghidra_scripts/*.java'
```

There's a `targetExclude` entry that we should use here instead. I think it should look like this
```suggestion
        target 'src/*/java/**/*.java', 'ghidra_scripts/*.java'
        targetExclude('src/main/java/mserialize/**/*.java')
```